README.md

```diff
@@ -442,6 +458,44 @@ export const action = async ({ request }) => {
 
 To better understand how to structure your data, refer to [qs documentation](https://github.com/ljharb/qs#parsing-objects)
 
+## Error Utilities
+To improve DX when dealing with errors we do export a couple of utilities.
+
+### errorsByName
```

Comment:
Shouldn't this be called `errorByName` in singular form?

Reply:
Or maybe `errorOn`?

Reply:
Hehe, actually, I think these helpers should be referring to `messages` and not `errors`, because we're returning messages and not the error objects.

Maybe `messagesFor` and `messagesForSchema`?

Reply:
I'd rather have both words to easily know what they refer to.. `errorMessagesFor`, `errorMessagesForSchema`

Reply:
I settled with your suggestion @gustavoguichard